### DISCUSSION TITLE
Database Format *.csv

### DISCUSSION THREAD

**Discussion by kinichahau2023** [RaBa64/OpenDTU#16] [Ideas]
Neue Idee /Vorschlag:
Ralf, schau doch mal, ob die Database auch im *.csv Format abgespeichert/downgeloaded werden kann. Damit wäre sie idealerweise direkt weiter einlesefähig und auswertbar in Excel (oder ähnlichen Programmen). 
Gerade für die, die kein übergeordneten System haben, wäre das eine tolle Option.

**Comment by RaBa64** [comment]
Excel kann mit Bordmittel schon JSON Daten über eine URL einlesen:
https://psdata.un.org/howto/excel_json

**Reply by kinichahau2023** [reply]
Dafür bin ich offenbar zu blöd mit meinem alten Excel 2010... hab zwar diese notwendige Erweiterung Power query schon gefunden, runtergeladen und installiert, aber so einfach ist es eben dann immer noch nicht!
Ok, danke erstmal ... vielleicht kriege ich es ja doch irgendwann irgendwie hin... 
(*.csv wäre allerdings immer noch schön ;-)
